### PR TITLE
ENH: Export ITK build trees on Linux

### PR DESCRIPTION
**Pull request by thewtex**
This should be merged after:

  http://review.source.kitware.com/#/c/22367

### CODE REVIEW

File: scripts/internal/manylinux-build-wheels.sh [hunk 1]
```diff
@@ -67,12 +67,19 @@ for PYBIN in "${PYBINARIES[@]}"; do
...
     ${PYBIN}/python setup.py bdist_wheel --build-type MinSizeRel -G Ninja -- \
       -DITK_SOURCE_DIR:PATH=/work/standalone-${arch}-build/ITK-source \
+      -DITK_BINARY_DIR:PATH=${build_path} \
       -DPYTHON_EXECUTABLE:FILEPATH=${PYTHON_EXECUTABLE} \
       -DPYTHON_INCLUDE_DIR:PATH=${PYTHON_INCLUDE_DIR} \
       -DPYTHON_LIBRARY:FILEPATH=${PYTHON_LIBRARY}
     ${PYBIN}/python setup.py clean
+    find $build_path -name '*.cpp' -delete -o -name '*.xml' -delete
+    rm -rf $build_path/Wrapping/Generators/castxml*
+    find $build_path -name '*.o' -delete
```
Comment (jcfr) [inline scripts/internal/manylinux-build-wheels.sh]: Makes sense.

That said, I wonder what could be done to have `python setup.py clean` really removing all artifacts ...

Comment (jcfr) [inline scripts/internal/manylinux-build-wheels.sh]: Since you `rm -rf $build_path` just above, it this really needed ?

Comment (thewtex) [inline scripts/internal/manylinux-build-wheels.sh]: This is so the build can be inspected / used between builds.

Comment (jcfr) [inline scripts/internal/manylinux-build-wheels.sh]: Based on your conversation, this is done here to reduce the size of the packaged archive. 

May be a comment could be added ?

Comment (thewtex) [inline scripts/internal/manylinux-build-wheels.sh]: Yeap, sending a new revision.

**Pull request merged**